lab_1/main.py

```diff
@@ -51,7 +80,14 @@ def create_language_profile(language: str, text: str, stop_words: list) -> dict
     :param stop_words: a list of stop words
     :return: a dictionary with three keys – name, freq, n_words
     """
-    pass
+    if isinstance(language, str) and isinstance(text, str) and isinstance(stop_words, list):
+        profile_lang = {'name': language,
+                        'freq': calculate_frequencies(remove_stop_words(tokenize(text),
+                                                                        stop_words)),
+                        'n_words': len(calculate_frequencies(remove_stop_words(tokenize(text),
```

Comment:
imagine that you have an extremely large text
tokenization and everything can take time
do you want to wait two times longer?

Reply:
Rewrote function so I have only one calculate_frequiencies function